src/trezor.js

```diff
@@ -123,9 +127,7 @@ try {
 } catch(e) {
   // We hit this if we run this code outside of a browser, for example
   // during unit testing.
-  /* eslint-disable no-process-env */
-  if (process.env.NODE_ENV !== 'test') {
-    /* eslint-enable */
+  if (env_variables.NODE_ENV !== 'test') {
```

Comment:
I wonder if this will actually quiet the below error in testing now. Could be that it wasn't working because of the direct access issue in the comment above.

Reply:
I only noticed the direct access issue when trying to import this into react, otherwise it works fine (also tests pass, w00t!)

Reply:
Oops my previous comment was a quick reply with other stuff going on, so didn't quite get what you were saying.

Ya - tests pass now and I don't see any error messages. 
See before:

![Screen Shot 2022-02-24 at 12 11 17 AM](https://user-images.githubusercontent.com/10471648/155484443-5f2c7092-899b-4204-bd66-eccb89a8594b.png)

and after this change:

![Screen Shot 2022-02-24 at 12 11 46 AM](https://user-images.githubusercontent.com/10471648/155484446-1da6ee21-de92-4f7a-a874-1ab85b10cd72.png)

Reply:
fantastic. good snipe